solar_system.rb

```diff
@@ -0,0 +1,34 @@
+class SolarSystem
+
+  attr_reader :star_name, :planets
+
+
+  def initialize(star_name)
+    @star_name = star_name
+    @planets = []
+  end
+
+  def add_planets (planet)
```

Comment:
This is nitpicking, but you shouldn't have a space between the method name and the parentheses. Your code will still run just fine though!